CMSgov/onemac
# OY2-10434: style updates to Group & Division selection

Story: https://qmacbis.atlassian.net/browse/OY2-10434
Endpoint: https://d3huklncg3vj6n.cloudfront.net/

### Details

Some minor adjustments to the Group & Division selection screen.

### Changes

- Change heading to "Select a Group and Division"
- Alphabetize dropdown elements
- Add full names to options and bold abbreviations per Figma
- Remove OACT per updated list from CMS

### Implementation Notes

N/A

### Test Plan

1. Log in as `cmsreviewerunregistered` and select CMS Reviewer. Verify that the specified cosmetic changes have been applied.
2. Submit the form and verify that it still works.
